frontend/src/helpers/endpoint.ts

```diff
@@ -38,12 +38,25 @@ export default async function api(config: ApiConfig): Promise<ApiResponse> {
       ? process.env.ENDPOINT_PROD
       : process.env.ENDPOINT_DEV;
 
-  // Configure local service port.
-  let servicePort = getServicePorts(config.service);
+  // Configure local service port based on the 'service' property in the configuration.
+  let servicePort = ":";
+  switch (config.service) {
+    case SERVICE.QUESTION:
+      servicePort += process.env.ENDPOINT_QUESTION_PORT || "";
+      break;
+    case SERVICE.USER:
+      servicePort += process.env.ENDPOINT_USER_PORT || "";
+      break;
+    default:
+      servicePort = "";
+      break;
+  }
```

Comment:
oh, I extracted the switch statement into `getServicePorts()` function already, so you can call it instead of impl the switch again.